### DISCUSSION TITLE
[Feature] Update person names in photo view

### DISCUSSION THREAD

**Discussion by ryanjdillon** [immich-app/immich#11848] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### User story
As a user, I would like to easily add/edit the names of people identified in a photo.

### Possible acceptance criteria
* When in the photo view, add an option to add/edit the name of a person shown in the person pane.
* When a person view is navigated to from a photo, return to the photo when the back button is pressed.

### Description
Using group photos seems like it would be a useful UI for naming identified persons, or just from any photo, but this 1) requires that you click the person to go to the person view, rather than adding the name in situ while viewing a photo, 2) After navigating to the person view, clicking back returns you to a gallery view and not the photo view.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
> When in the photo view, add an option to add/edit the name of a person shown in the person pane.

You can get to the person page with one click, so I don't think this is necessary, and it'd probably clutter the UI.

> When a person view is navigated to from a photo, return to the photo when the back button is pressed.

This already works
